README.md

```diff
@@ -54,6 +54,32 @@ import * as Turbo from '@hotwired/turbo'
 +TurboPower.initialize(Turbo.StreamActions)
 ```
 
+### Installation on a stock Rails 7 install, which utilizes [importmaps](https://github.com/rails/importmap-rails)
+
+On a stock install of Rails 7 modify the following files:
+
+**config/importmaps.rb**
+```diff
+pin "application", preload: true
+- pin "@hotwired/turbo-rails", to: "turbo.min.js", preload: true
++ pin "@hotwired/turbo", to: "turbo.min.js", preload: true
+pin "@hotwired/stimulus", to: "stimulus.min.js", preload: true
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js", preload: true
+```
```

Comment:
I don't think this should be necessary. It should also be possible to just use the `turbo-rails` package. 

So I think we can keep this as `turbo-rails` and just adjust the import in the `application.js` instead.

Otherwise if you ran `bin/importmap pin turbo_power` yourself you would need to update the file like this:

```diff
- pin "@hotwired/turbo", to: "https://ga.jspm.io/npm:@hotwired/turbo@7.2.2/dist/turbo.es2017-esm.js"

+ pin "turbo_power", to: "https://ga.jspm.io/npm:turbo_power@0.1.4/dist/index.js"
+ // or use the bundled script shipped with the `turbo_power` gem
+ pin "turbo_power", to: "turbo_power.js"
```

Reply:
Ok right. I added the `pin` command and the fact that the `pin "@hotwired/turbo", to: "https://ga.jspm.io/npm:@hotwired/turbo@7.2.2/dist/turbo.es2017-esm.js"` needs to be removed.

But yes, the `pin "@hotwired/turbo", to: "turbo.min.js", preload: true` is necessary.